.github/ISSUE_TEMPLATE/release_template.md

```diff
@@ -18,31 +18,44 @@ For OEM releases, keep the OEM Release template and remove the Open Release one
 
  - [ ] [DOC] Ping in #documentation-internal about the new release
  - [ ] [GIT] Create branch `release/M.m.p` in owncloud/android from master
+ - [ ] [GIT] Rebase `release/M.m.p` against `stable` in owncloud/android
+ - [ ] [GIT] Create branch `release/x.y.z` in owncloud/android-library from master
+ - [ ] [GIT] Rebase `release/x.y.z` against `stable` in owncloud/android-library
  - [ ] [DEV] Update version number and name in build.gradle in owncloudApp module
  - [ ] [DEV] Update [SBOM](https://cloud.owncloud.com/f/6072870)
  - [ ] [DIS] Create a folder for the new version like `M.m.p_YYYY-MM-DD` inside the `changelog` folder
  - [ ] [DIS] Move all changelog files from the unreleased folder to the new version folder
  - [ ] [DIS] Update screenshots, if needed, in README.md
  - [ ] [DIS] Add ReleaseNotes replacing `emptyList` with `listOf` and adding inside `ReleaseNote()` with String resources
+ - [ ] [DIS] Prepare post in central.owncloud.org ([`Category:News + Tag:android`](https://central.owncloud.org/tags/c/news/5/android))
  - [ ] [QA] Design Test plan
  - [ ] [QA] Regression Test execution
- - [ ] [GIT] Create and sign tag `oc-android-M.m.p` in HEAD commit of release branch, in owncloud/android
- - [ ] [GIT] Create and sign tag `x.y.z` in HEAD commit of release branch, in owncloud/android-library
  - [ ] [DIS] Generate final bundle from signed commit in owncloud/android
+ - [ ] [DIS] Upload release APK and bundle to internal owncloud instance
+ - [ ] [DEV] Code Review
```

Comment:
Maybe this fits better just after the adding ReleaseNotes step, since if there is anything to change there, it would be more consistent that the bundle used for QA (and even uploading to Google Play) references the same commit as the final bundle.

Reply:
i had also that idea, but, after QA (and also, release notes) bugs fixes could need another code review. It will trigger another loop in case code review changes important code and more QA is required, then new bugs could raise up, more fixes and another new review etc...

So, finally, i left the code review after the first loop (release notes, fixed and everything) because no matter where you place the "code review" step, always you may find another code review.

Is this ok?

Reply:
Well, if the final approval is implicit in the QA process as you mentioned, then I think it should go before anyways. Although you need to start another loop, I think CR should be always made before QA.
Just thinking on going step by step reading this template, I think it would make sense finding the CR step before QA, both when it is only one loop or several ones. But this is just my point of view 👀

Reply:
Well, if the final approval is implicit in the QA process as you mentioned, then I think it should go before anyways. Although you need to start another loop, I think CR should be always made before QA.
Just thinking on going step by step reading this template, I think it would make sense finding the CR step before QA, both when it is only one loop or several ones. But this is just my point of view 👀

Reply:
Ok, i will move to before, and create another step `QA approval`, separating the aproval itself from the test execution. So, these two steps:

```
 - [ ] [DEV] Code Review
 - [ ] [QA] Regression Test execution
 ```

are in a loop till QA clicks the new one `QA approval`

Reply:
Nice! I like it! 👍